core/src/main/java/fr/sncf/osrd/api/pathfinding/constraints/ElectrificationConstraints.java

```diff
@@ -9,9 +9,7 @@
 import fr.sncf.osrd.train.RollingStock;
 import fr.sncf.osrd.utils.graph.Pathfinding;
 import fr.sncf.osrd.utils.graph.functional_interfaces.EdgeToRanges;
-import java.util.Collection;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
```

Comment:
`import ...*` is generally not a good practice. Try to configure your editor so it does not do so (in intellij for ex, there's a max number of imports after which it switches to *, increase it to say 999).

Reply:
You're right